src/components/application_manager/include/application_manager/commands/mobile/reset_global_properties_request.h

```diff
@@ -108,6 +108,15 @@ class ResetGlobalPropertiesRequest : public CommandRequestImpl {
       application_manager::ApplicationSharedPtr const app);
 
   /*
+   * @brief Prepare result code and result for sending to mobile application
+   * @param result_code contains result code for sending to mobile application
+   * @param response_info contains info for sending to mobile applicaion
+   * @return result for sending to mobile application.
+   */
+  bool PrepareResponseParameters(mobile_apis::Result::eType& result_code,
```

Comment:
@dcherniev  Please rename parameters according to this document: https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide